### PR TITLE
Update "try Cognitive Services for free" link

### PR DESCRIPTION
**Pull request by v-jaswel**
## Purpose
Update "try Cognitive Services for free" links.
Also remove references to "westus" region used by free trial subscriptions.

## Pull Request Type
What kind of change does this Pull Request introduce?

```
[ ] Bugfix
[ ] Feature
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[ ] Documentation content changes
[ ] Other... Please describe:
```

### CODE REVIEW

File: quickstart/cpp/windows/intent-recognition/README.md [hunk 1]
```diff
@@ -4,7 +4,7 @@ This sample demonstrates how to recognize intent with C++ using the Speech SDK f
 
 ## Prerequisites
 
-* A LUIS account. You can get one for free through the [LUIS portal](https://www.luis.ai/home).
+* A LUIS account. [Create a new Azure account, and try Cognitive Services for free.](https://azure.microsoft.com/free/cognitive-services/)
```
Comment (ChrisHMSFT) [inline quickstart/cpp/windows/intent-recognition/README.md:7]: @v-jaswel  we should remove this change since it's referring to creating a LUIS account not a subscription. thanks!

File: quickstart/javascript/browser/intent-recognition/README.md [hunk 2]
```diff
@@ -7,7 +7,7 @@ This sample shows how to recognize intent with the Speech Service using the Spee
 
 ## Prerequisites
 
-* A LUIS account. You can get one for free through the [LUIS portal](https://www.luis.ai/home).
+* A LUIS account. [Create a new Azure account, and try Cognitive Services for free.](https://azure.microsoft.com/free/cognitive-services/)
```
Comment (ChrisHMSFT) [inline quickstart/javascript/browser/intent-recognition/README.md:10]: this change should probably be reverted. the original is about a LUIS account not an azure subscription. thanks!

File: samples/unity/speechrecognizer/Assets/SpeechSDKSample/Scripts/IntentRecognition.cs [hunk 3]
```diff
@@ -71,9 +71,8 @@ public class IntentRecognition : MonoBehaviour {
     private void Awake()
     {
         // IMPORTANT INFO BEFORE YOU CAN USE THIS SAMPLE:
-        // Get your own Cognitive Services LUIS subscription key for free by following the
-        // instructions under the section titled 'Get LUIS key' in the article found at
-        // https://docs.microsoft.com/azure/cognitive-services/luis/luis-get-started-cs-get-intent.
+        // If you do not have a LUIS subscription key, you can visit the Microsoft Cognitive Services Web site
+        // at https://azure.microsoft.com/free/cognitive-services/, create a new Azure account, and try Cognitive Services for free.
         // Use the inspector fields to manually set these values with your subscription info.
```
Comment (dargilco) [inline samples/unity/speechrecognizer/Assets/SpeechSDKSample/Scripts/IntentRecognition.cs:76]: Revert this change.. it's about LUIS key, not speech key

File: samples/unity/speechrecognizer/Assets/SpeechSDKSample/Scripts/SpeechRecognition.cs [hunk 4]
```diff
@@ -71,8 +71,8 @@ public class SpeechRecognition : MonoBehaviour
     private void Awake()
     {
         // IMPORTANT INFO BEFORE YOU CAN USE THIS SAMPLE:
-        // Get your own Cognitive Services Speech subscription key for free at the following
-        // link: https://docs.microsoft.com/azure/cognitive-services/speech-service/get-started.
+        // If you do not have a LUIS subscription key, you can visit the Microsoft Cognitive Services Web site
+        // at https://azure.microsoft.com/free/cognitive-services/, create a new Azure account, and try Cognitive Services for free.
         // Use the inspector fields to manually set these values with your subscription info.
```
Comment (dargilco) [inline samples/unity/speechrecognizer/Assets/SpeechSDKSample/Scripts/SpeechRecognition.cs:76]: Revert his change... it's about LUIS key, not speech key

File: quickstart/java/android/intent-recognition/README.md [hunk 5]
```diff
@@ -9,7 +9,7 @@ See the [accompanying article](https://docs.microsoft.com/azure/cognitive-servic
 
 ## Prerequisites
 
-* A LUIS account. You can get one for free through the [LUIS portal](https://www.luis.ai/home).
+* A LUIS account. [Create a new Azure account, and try Cognitive Services for free.](https://azure.microsoft.com/free/cognitive-services/)
 * A new or existing LUIS app - [create LUIS app](https://docs.microsoft.com/azure/cognitive-services/speech-service/quickstarts/intent-recognition#create-a-luis-app-for-intent-recognition)
```
Comment (dargilco) [inline quickstart/java/android/intent-recognition/README.md:13]: Revert this change.. it's about LUIS account, not Speech key

**Comment by dargilco** [comment]
After looking at both links (the existing one and the suggested new one in this PR), I disagree with the change. We need a link that describes both how to get an Azure account and how to create a speech resource and get the key and key region. The existing link does just that: https://docs.microsoft.com/azure/cognitive-services/speech-service/get-started . The new suggested link is a generic Cognitive Service link. It does not tell you how to create a Speech resource on Azure. Closing this PR.
